src/components/NowPlaying.vue

```diff
@@ -15,6 +15,7 @@
                 </template>
 
                 <b-dropdown-item-button @click.stop="addToLibrary(nowPlayingItem)">Add to library</b-dropdown-item-button>
+                <b-dropdown-item-button @click.stop="goToAlbum(nowPlayingItem)">Go to album</b-dropdown-item-button>
```

Comment:
If you click on the 3-dots next to any song in the song list, you'll see that there is a separator between the "authenticated" actions and the "unauthenticated" actions. I would like to see that convention continued. See https://github.com/zachomedia/apple-music-webplayer/blob/master/src/components/Songs.vue#L40

Also, the `v-if="isAuthorized"` attribute from the `b-dropdown` element needs to be removed and added to the "Add to library" button (as seen in the Songs component). Without this, you can't access to "Go to album" while logged out.

Reply:
I made this change locally, but it looks like the "assets" propery of the MediaItem is empty when logged out. Since the data isn't there, I'm not sure how to properly link to the album id when logged out (without searching for it). Since the method I chose needs authentication, I left the dropdown how it is for now (don't want an empty dropdown when not authenticated).

Reply:
I think that's the best way to handle it for now.